vspace/src/main/java/edu/asu/diging/vspace/web/publicview/ExhibitionSequencesController.java

```diff
@@ -54,12 +60,20 @@ public String sequence(Model model, @PathVariable("sequenceId") String sequenceI
         if (sequenceExist==null) {
             throw new SequenceNotFoundException(sequenceId);
         }
-        
+
         List<ISlide> slides = sequenceManager.getSequence(sequenceId).getSlides();
         if (slides.size() == 0) {
             throw new SlidesInSequenceNotFoundException();
         }
+
+        if(choiceId==null) {
+            choiceId=sequenceId;
```

Comment:
so choiceId is actually sequence id? bad variable name

Reply:
Yes, but we want to store currentSequenceId and we lose that info when we click on choice from branching point. currentSequenceID, sequenceID would be confusing so I thought of using choiceID. Should I call that currentSequenceID?

Reply:
ignore, removed this block of code.